docs/csharp/language-reference/operators/remainder-operator.md

```diff
@@ -1,28 +1,32 @@
 ---
 title: "% Operator (C# Reference)"
-ms.date: 07/20/2015
+ms.date: 04/03/2018
 ms.prod: .net
 ms.technology: 
   - "devlang-csharp"
 ms.topic: "article"
 f1_keywords: 
   - "%_CSharpKeyword"
 helpviewer_keywords: 
-  - "modulus operator [C#]"
+  - "remainder operator [C#]"
   - "% operator [C#]"
 ms.assetid: 3b74f4f9-fd9c-45e7-84fa-c8d71a0dfad7
 caps.latest.revision: 15
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # % Operator (C# Reference)
-The `%` operator computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators.  
+The remainder operator (`%`) computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators. 
```

Comment:
This is pre-existing text so it's a separate issue but it's worth pointing out that the text "all numeric types have predefined X operators" is kind of bogus. Small numeric types (byte, short etc.) do not have such operators.

Reply:
The C# spec says that there is a predefined `%` operator for `int`, `uint`, `long`, `ulong`, `float`, `double`, and `decimal`. 

`byte`, `short`, `ushort`, are all implicitly convertible to `int`, so the `%` operator works and does what's expected:

```csharp
byte b1 = 5;
byte b2 = 2;
int b3 = b1 % b2; // b1 and b2 are both converted to `int`, result is `int`
```

If you were to change it, I'd suggest "All numeric types support the remainder operator". It makes the same point, without disagreeing with the spec.

Reply:
@mikedn @BillWagner What about for other operations? Does `b1 + b2` give `byte`? Multiplication? Division? Or is it just Remainder missing an operator definition for `byte`?